src/portable/windows.rs

```diff
@@ -866,6 +866,21 @@ pub fn revert(options: &options::Revert) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub fn get_ui_token(name: &str) -> anyhow::Result<String> {
+    let wsl = try_get_wsl()?;
+    let token = wsl.edgedb()
+        .arg("instance")
+        .arg("_ui_token")
+        .arg(name)
+        .get_stdout_text()?;
+    Ok(token)
+}
+
+pub fn ui_token(name: &str) -> anyhow::Result<()> {
+    println!("{}", get_ui_token(name)?);
```

Comment:
nit: this looks like will print double linefeed? (one from the captured text and one from println here).